Automattic/themes
# Quadrat: add illustrations as starter content

#### Changes proposed in this Pull Request:

This PR bundles illustrations as starter content. 

**Testing Instructions**
- Create a fresh site
- Activate Quadrat
- Verify the illustrations appear in the media library

#### Related issue(s):

Partially addresses #4211
